src/code/z_message_PAL.c

```diff
@@ -3172,7 +3172,7 @@ void Message_Update(PlayState* play) {
                     R_TEXTBOX_TEXHEIGHT = 512;
                 } else {
                     Message_GrowTextbox(msgCtx);
-                    Audio_PlaySoundIfNotInCutscene(0);
+                    Audio_PlaySoundIfNotInCutscene(NA_SE_PL_WALK_GROUND - SFX_FLAG);
```

Comment:
hey it's the same jank block of code with the `R_SCENE_CAM_TYPE` stuff :PepeGa:

I guess it does nothing because of `- SFX_FLAG`, it plays sound for one frame (notice `msgMode = MSGMODE_TEXT_BOX_GROWING` below), idk if it can be heard at all

Reply:
There's seems to be a lot of weirdness when it comes to sfxId = 0. In `z_message` alone, there's plenty of instances of `Audio_PlaySoundGeneral(0, ...)`, and there's an early return for if the `sfxId = 0`:
https://github.com/engineer124/oot/blob/eb462b298e54b402066dced6ad2629c1514f188e/src/code/code_800F7260.c#L233-L235

I'm hesitant to say it was intended as `NA_SE_PL_WALK_GROUND` and would leave this as 0 until it can be worked out exactly what happens with `sfxId = 0`

Reply:
Oh really, weird
Can I slap a TODO here? like `// TODO: this may be NA_SE_PL_WALK_GROUND - SFX_FLAG, or not, investigate sfxId=0`

Reply:
Sure